Shared/Preferences/PreferencesModel.swift

```diff
@@ -1,67 +1,54 @@
 import SwiftUI
 
+enum Appearance: Int {
+    case system = 0
+    case light = 1
+    case dark = 2
+}
+
 class PreferencesModel: ObservableObject {
     private let defaults = UserDefaults.shared
 
-    /* We're stuck dropping into AppKit/UIKit to set light/dark schemes for now,
-     * because setting the .preferredColorScheme modifier on views in SwiftUI is
-     * currently unreliable.
-     *
-     * Feedback submitted to Apple:
-     *
-     * FB8382883: "On macOS 11β4, preferredColorScheme modifier does not respect .light ColorScheme"
-     * FB8383053: "On iOS 14β4/macOS 11β4, it is not possible to unset preferredColorScheme after setting
-     *              it to either .light or .dark"
-     */
-
-    #if os(iOS)
-    @available(iOSApplicationExtension, unavailable)
-    var window: UIWindow? {
-        guard let scene = UIApplication.shared.connectedScenes.first,
-              let windowSceneDelegate = scene.delegate as? UIWindowSceneDelegate,
-              let window = windowSceneDelegate.window else {
-            return nil
```

Comment:
Moving this into PreferencesView, to fix the “stored property is unavailable in application extensions for iOS” error in Xcode 15/iOS 17.